docs/user_guide/authenticate.md

```diff
@@ -1,11 +1,139 @@
+
+## Login Manually
+
+If you have not created a `.netrc` file or `EARTHDATA_USERNAME` and `EARTHDATA_PASSWORD`, you can use the following approach to login.
+
+```
+>>> import earthaccess
+
+>>> auth = earthaccess.login()
+Enter your Earthdata Login username: your_username
+Enter your Earthdata password: 
+```
+
+You don't need to assign the result of `earthaccess.login()` to a variable but doing so enables access session information.  These are discussed in [Accessing Session Information]().
+
+Setting `strategy=interactive` will force a manual login.
+
+## Login using a `.netrc`
+
+### Creating a `.netrc` file
+
+#### Using `earthaccess.login` to create a `.netrc` file
+
+You can use `earthaccess.login` to create a `.netrc` for you.
+```
+earthaccess.login(persist=True)
+```
+You will then be prompted for your Earthdata Login username and password.  A `.netrc` (or `_netrc`) file will be created automatically.
+
+#### Manually creating a `.netrc` file for Earthdata Login Credentials
+
+=== "MacOS"
+    Type the following on your command line, replacing `<username>` and `<password>` with your
+    Earthdata Login credentials.
+    
+    ```
+    echo "machine urs.earthdata.nasa.gov login <username> password <password>" >> $HOME/.netrc
+    chmod 600 $HOME/.netrc
+    ```
+
+=== "Linux"
+    Type the following on your command line, replacing `<username>` and `<password>` with your
+    Earthdata Login credentials.
+    
+    ```
+    echo "machine urs.earthdata.nasa.gov login <username> password <password>" >> $HOME/.netrc
+    chmod 600 $HOME/.netrc
+    ```
+
+=== "Windows"
+    In a `CMD` session, create a `%HOME%` environment variable.  The following line
+    creates `%HOME%` from the path in `%USERPROFILE%`, which looks something like
+    `C:\Users\"username"`.
+    ```
+    setx HOME %USERPROFILE%
+    ```
+    You now need to create a `_netrc` file in `%HOME%`.
+    ```
+    echo "machine urs.earthdata.nasa.gov login <username> password <password>" >> %HOME%\_netrc
+    ```
+
+## Login using environment variables
+
+Alternatively, Earthdata Login credentials can be created as environment variables EARTHDATA_USERNAME and EARTHDATA_PASSWORD.
+
+=== "MacOS"
+    If you want to set the environment variables for the current shell session, type the following on the command line.
+    ```
+    export EARTHDATA_USERNAME="username"
+    export EARTHDATA_PASSWORD="password"
+    ```
+    If you want to set these environmental variables permanently, add these two lines to your `.profile` file.
+    
+
+=== "Linux"
+    If you want to set the environment variables for the current shell session, type the following on the command line.
+    ```
+    export EARTHDATA_USERNAME="username"
+    export EARTHDATA_PASSWORD="password"
+    ```
+    If you want to set these environmental variables permanently, add these two lines to your `.bashrc` file.
```

Comment:
`.bashrc` for Linux and `.profile` for Mac won't always be correct. I'm not very familiar with zsh yet, but I believe it uses a `.zprofile` file? Bash users can permanently set envvars in `.profile` as well, but that will only work for "login" shells. There isn't really a one-size-fits-all answer here, it depends on the shell.

Reply:
So should we just say "add [these] to the appropriate configuration files for your operating system."

Reply:
I think that makes sense as a fallback instruction; e.g.:

> If you use `bash` and would like to set these environment variables permanently, add these two lines to your `~/.profile` file: ... code ... . Otherwise, use the recommended method to persistently set environment variables for whichever shell you use.

Something like that?